lisa/messages.py

```diff
@@ -144,6 +144,7 @@ class PerfMessage(MessageBase):
         "unknown",
         "nvme",
         "premiumssd",
+        "ultradisk",
```

Comment:
We have Enums for Disk Type in several other places. Can we resuse code so that less changes need to be made in the future?

Reply:
The other place where I found the disk types declared are "schema.DiskType". But these the names are a bit different compared to ones we have here. The disk types used in messages.py are used in db table. Changing them to different ones might break multiple storage perf powerbi reports. @LiliDeng please share your thoughts.

Reply:
Okay, I think it is okay to keep the current implementation if it is representing values in a different database.